src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -31,28 +33,24 @@ protected internal override void Setup(FeatureConfigurationContext context)
             var transportInfrastructure = context.Settings.Get<TransportInfrastructure>();
 
             var unicastRoutingTable = context.Settings.Get<UnicastRoutingTable>();
+            unicastRoutingTable.SetLogChangeAction(GetLogAction<UnicastRoutingTable>());
```

Comment:
As far as I can see, the only reason for this is to test the logs, is that right? Is this really worth the effort?